packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -27,7 +27,7 @@ var get = Ember.get;
   @class BuildURLMixin
   @namespace DS
 */
-var BuildURLMixin = Ember.Mixin.create({
+export default Ember.Mixin.create({
```

Comment:
really dislike this style as it makes the code less greppable and you also have to undo it later if you have to call reopen or reopenClass.